### ISSUE TITLE
Obtain latest minor version for a given node major version

### ISSUE THREAD

**Issue by claudiopro** [nodejs/help#428]
What's the most reliable and easy way to get the latest minor version for a given node major version?  I'm [writing a tool](https://github.com/claudiopro/travisci-node-update) that updates Travis CI test targets to latest available minor versions.

Thank you! 😊

**Comment by addaleax** [comment]
There are the `index.json` and `index.tab` in https://nodejs.org/download/release/ which will probably help you?

But just to make sure, you do know that just specifying
```yml
node_js:
  - "7"
```
will make `nvm` automatically select the latest v7.x version of Node, right? Is there any reason you are particularly interested in minor version bumps?

**Comment by claudiopro** [comment]
Thanks @addaleax, that's exactly what I needed 😄 

Yes the tool should not modify the detail of declared version, and should preserve it after identifying the correct replacement, e.g.:

### Before
```yaml
node_js:
  - '7'
  - '7.2'
  - '6.4.3'
  - '6.9'
  - '5'
  - '5.9.2'
  - '5.12'
  - '4'
  - '4.6'
  - '4.4.3'
```

### After
```yaml
node_js:
  - '7'
  - '7.3'
  - 6.9.2
  - '6.9'
  - '5'
  - 5.12.0
  - '5.12'
  - '4'
  - '4.7'
  - 4.7.0
```

Thank you so much! I released `v1.0.0`: [travisci-node-update](https://www.npmjs.com/package/travisci-node-update)
